samples/performance-counters/PerfCounters_1/Sample/packages.config

```diff
@@ -1,5 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="NServiceBus" version="6.2.1" targetFramework="net461" />
-  <package id="NServiceBus.Metrics.PerformanceCounters" version="1.0.0" targetFramework="net461" />
+  <package id="NServiceBus.Metrics" version="1.0.0-alpha0065" targetFramework="net461" />
+  <package id="NServiceBus.Metrics.PerformanceCounters" version="1.0.1-metrics-dependen0001" targetFramework="net461" />
```

Comment:
not that  have an issue with this. but note it looks like a PR nuget. was this intended?